src/options/options.css

```diff
@@ -1,6 +1,11 @@
+:root {
+  --gray: rgba(12, 12, 13, 0.15);
```

Comment:
Is this an official Photon variable?

If so, then you could also [include it here](https://github.com/rugk/offline-qr-code/blob/master/src/common/variables.css). (Or if it is useful in other contexts.)

Note you'd better leave it as it is, if it is no official variable.